jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/DefaultControllerEnricherTest.java

```diff
@@ -91,43 +97,24 @@ public void checkDefaultReplicaCount() {
         assertImagePullPolicy(builder, "IfNotPresent");
     }
 
-    @Test
-    public void create_withImagePullPolicyConfigured_shouldUseConfiguredImagePullPolicy() {
-        // Given
-        properties.put("jkube.enricher.jkube-controller.pullPolicy", "Never");
-
-        // When
-        KubernetesListBuilder builder = enrich();
-
-        // Then
-        assertReplicas(builder, 1);
-        assertImagePullPolicy(builder, "Never");
-    }
-
-    @Test
-    public void create_withImagePullPolicyProperty_shouldUseConfiguredImagePullPolicy() {
+    @DisplayName("create with")
+    @ParameterizedTest(name = "configured {0}, should use configured {0} ({2})")
```

Comment:
This description makes no sense.
Also, once more, you've combined far too many tests
You can combine `create_withImagePullPolicyConfigured_shouldUseConfiguredImagePullPolicy` and `create_withImagePullPolicyProperty_shouldUseConfiguredImagePullPolicy` and talk about the resulting pull policy.

The other one you need to leave as is, or combine by providing all of the property combinations that affect the replica count outcome.